### DISCUSSION TITLE
Water cooling/pump support

### DISCUSSION THREAD

**Discussion by atomdeathstroke** [Rem0o/FanControl.Releases#529] [Ideas]
Any chance you can add settings for better controlling water pumps?  Those are much higher RPM and having a flow control method for popular pumps like DDC and D5 would be dope!

thanks!

**Comment by Rem0o** [comment]
Well they act just like fans as they are plugged into a normal header, with RPM and everything. What's missing?

**Reply by atomdeathstroke** [reply]
Mostly just a way to categorize it separately from fans.  Even though much of the functionality is the same, the categorization of it is different.  Also some builds now are using separate loops and pumps for GPU/CPU.  I think just another visual separation adds that much more customization and "showmanship" for this awesome software.

**Reply by Rem0o** [reply]
Just so you know, you can rename cards, and also move them by dragging and dropping within its category.

**Reply by atomdeathstroke** [reply]
That's what I'm saying, a new category for pumps.
